src/main/java/com/tstepnik/planner/domain/task/TaskDTO.java

```diff
@@ -0,0 +1,53 @@
+package com.tstepnik.planner.domain.task;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Size;
+
+public class TaskDTO {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotEmpty
+    @Size(min=3)
+    private String description;
+
+    @Enumerated(EnumType.STRING)
+    private Importance importance;
+
+    private boolean isDone;
```

Comment:
Also wroth making this Boolean since DTO will be accepted from the client.

Reply:
But Boolean needs more memory and i don't see any proses of it.ALso I need to set object befoer creating task because of null.